website/src/documentation/sdks/python-dependencies.md

```diff
@@ -26,460 +26,41 @@ behavior in the service. If you are using any of these packages in your code, be
 aware that some libraries are not forward-compatible and you may need to pin to
 the listed versions that will be in scope during execution.
 
-<p>To see the compile and runtime dependencies for your Beam SDK version, expand
-the relevant section below.</p>
+Dependencies for your Beam SDK version are listed in `setup.py` in the Beam repository. To view them, perform the following steps:
 
-<details><summary markdown="span"><b>2.17.0</b></summary>
+1. Open `setup.py`.
 
-<p>Beam SDK for Python 2.17.0 has the following compile and runtime dependencies.</p>
+    ```
+    https://raw.githubusercontent.com/apache/beam/v<version_number>/sdks/python/setup.py
+    ```
+    
+    Replace `<version_number>` with the major.minor.patch version of the SDK. For example, <https://raw.githubusercontent.com/apache/beam/v2.18.0/sdks/python/setup.py>{:target="_blank"} will provide the dependencies for the 2.18.0 release.
```

Comment:
I looked at the staged website: I think the documentation may be more user-friendly and the link we give in "For example" points to a place in the file where requirements are defined instead of the beginning of the file.
We could use 'https://github.com/apache/beam/...' instead of raw URL for this purpose. What do you think? 

Same about BeamModulePlugin reference.